R/dv8.R

```diff
@@ -0,0 +1,153 @@
+#' Convert from sdsm.json to a dependency matrix in sdsm.dv8-dsm file
+#'
+#' @param dsmj_path path to sdsm.json
+#' @param dsmb_path path to save dv8-dsm file
+#' @export
+dv8_dsmj_to_dsmb <- function(dsmj_path, dsmb_path){
+  # Expand paths (e.g. "~/Desktop" => "/Users/someuser/Desktop")
+  dsmj_path <- path.expand(dsmj_path)
+  dsmb_path <- path.expand(dsmb_path)
+
+  # Run system2 command
+  system2('dv8-console', args=c('core:convert-matrix', '-outputFile', dsmb_path, dsmj_path), stdout=FALSE, stderr=FALSE)
+}
+
+#' Merge matrices from dv8-dsm file to one new matrix in hsdsm.dv8-dsm
```

Comment:
Same documentation issue as above. Title should be consistent to Kaiaulu nomenclature. Body of function should be consistent to dv8 nomenclature.